src/controllers/statistics.js

```diff
@@ -0,0 +1,80 @@
+import models from '../models'
+import UserServices from '../services/user.service';
+
+const getStatistics = async function (req, res, next) {
```

Comment:
@Alice-Gatera this function is too long. You can create helper functions that fetch particular stats instead of doing all the operations in one and big function.

Reply:
@bdushimi thanks for the feedback, am going to work on it.